README.md

```diff
@@ -44,11 +44,11 @@ Steps for installation of OutcomeBasedCourse
 ```bash
 git clone https://github.com/GreatDevelopers/OutcomeBasedCourse
 ```
-1. Create a database for OutcomeBasedCourse
+2. Create a database for OutcomeBasedCourse
```

Comment:
This list can be corrected by adding three spaces before lines like:
   ```bash
   git clone https://github.com/GreatDevelopers/OutcomeBasedCourse
   ```

Reply:
Ohh nice.